ddl/ddl_api.go

```diff
@@ -3904,14 +3904,14 @@ func checkIsDroppableColumn(ctx sessionctx.Context, t table.Table, spec *ast.Alt
 	return true, nil
 }
 
-func checkDropVisibleColumnCnt(t table.Table, columnCnt int) error {
+func checkVisibleColumnCnt(t table.Table, addCnt, dropCnt int) error {
```

Comment:
How about combine `checkDropVisibleColumnCnt` and `checkAddColumnTooManyColumns`

Reply:
I prefer to separate them because they make different check.